README.md

```diff
@@ -45,6 +34,20 @@ to developers/contributors.
 - [X] Implement Crypto
 - [X] Create Messenger
 
+## Q&A:
```

Comment:
One could add a statement about security vs privacy vs anonymity, because tox provides the first two by default but anonymity requires extra effort. This issue comes up pretty frequently (ie. can my friend see my IP address?).